imports/ui/components/playlists/playlist_page.html

```diff
@@ -8,7 +8,7 @@ <h2 class='general__header'>
       {{/with}}
     {{else}}
       <h2 class='general__header'>
-        Sub Show playlist - {{showDateOfCurrent}}
+        Sub Show w/ {{djOfShow}} playlist - {{showDateOfCurrent}}
```

Comment:
Hi Dere,

I think that it should be along the lines of how it looks in the sidebar:

`{{showDateOfCurrent}} w/ {{djOfShow}}`

Actually, these subbed shows have names, which IIRC can vary: they can have the same name as the show in that timeslot but with a different than usual DJ, or they can have a newly made up name, and I think there was one more option. But I'm happy if it just says ie. `time w/ DJ`.